src/game.zig

```diff
@@ -325,5 +326,14 @@ pub fn update(deltaTime: f64) void {
 		defer Player.mutex.unlock();
 		Player.super.pos += movement*@as(Vec3d, @splat(deltaTime));
 	}
+
+	const biome = world.?.playerBiome.load(.seq_cst);
```

Comment:
.monotonic is enough. Sequential consistency just makes it more expensive for no reason